src/plugins/saved_objects_management/public/utils.ts

```diff
@@ -12,3 +16,35 @@ export function formatWorkspaceIdParams<
   }
   return others;
 }
+
+export function formatInspectUrl(
+  savedObject: SavedObjectWithMetadata,
+  useUpdatedUX: boolean,
```

Comment:
We can get useUpdatedUX by using the coreStart, seems we can remove this params.

Reply:
`useUpdatedUX` has been get and called in `saved_objects_table_page.tsx` file, I don't want to get it again.